libc/utils/docgen/cpio.yaml

```diff
@@ -0,0 +1,107 @@
+macros:
+  C_IRUSR:
+    c-definition: "Read by owner."
+    in-latest-posix: ''
+    value: "0000400"
```

Comment:
Thanks for adding these values, but they can be removed; hdrgen does not consume this field.  It won't break anything to retain them, but they just add noise for now. You can also remove the newlines between macros.